commit edc5be409c95cf13a30475a7d64bafa774db20d4
Author: Joe Walker <jwalker@mozilla.com>
Date:   Wed Nov 10 13:39:38 2010 +0000

    path that works both from file:// URLs and http:// URLs

diff --git a/editor.html b/editor.html
index 2d5c4b4..9f1aeec 100644
--- a/editor.html
+++ b/editor.html
@@ -134,7 +134,7 @@
 <script type="text/javascript" charset="utf-8">
 
 require(
-    {baseUrl: "../plugins/ace/lib"},
+    {baseUrl: "plugins/ace/lib"},
     [
         "ace/lib/event",
         "ace/editor",
